x-pack/legacy/plugins/security/public/lib/api_keys_api.ts

```diff
@@ -5,8 +5,7 @@
  */
 
 import { kfetch } from 'ui/kfetch';
-import { ApiKey, ApiKeyToInvalidate } from '../../common/model/api_key';
-import { INTERNAL_API_BASE_PATH } from '../../common/constants';
```

Comment:
__note:__ tbh I don't see much value in `INTERNAL_API_BASE_PATH` constant and it makes it harder to copy-paste URL to find where it's defined. But let me know if you prefer to have it.

Reply:
I'm in favor of removing this as well. I enjoy being able to search on full API paths in code